docs/source/CartesianConfig.rst

```diff
@@ -41,11 +41,11 @@ definitions.
 Variants
 ========
 
-A ‘variants’ stanza is opened by a ‘variants:’ statement. The contents
-of the stanza must be indented further right than the ‘variants:’
-statement. Each variant stanza or block defines a single dimension of
+A ‘variants’ block is opened by a ‘variants:’ statement. The contents
+of the block must be indented further right than the ‘variants:’
+statement. Each variant block defines a single dimension of
 the output array. When a Cartesian configuration file contains
-two variants stanzas, the output will be all possible combination's of
+two variants blocks, the output will be all possible combination's of
```

Comment:
OK, I don't have problems with either of them but block is probably more widely used...